src/screens/NewAnalytics/PaymentAnalytics/PaymentsProcessed/PaymentsProcessed.res

```diff
@@ -14,10 +15,17 @@ module TableModule = {
     }
     let tableBorderClass = "border-collapse border border-jp-gray-940 border-solid border-2 border-opacity-30 dark:border-jp-gray-dark_table_border_color dark:border-opacity-30"
 
-    let paymentsProcessed =
-      data
-      ->LogicUtils.getArrayDataFromJson(tableItemToObjMapper)
-      ->Array.map(Nullable.make)
+    let paymentsProcessed = switch data->getArrayFromJson([])->Array.get(0) {
+    | Some(val) => {
+        let valueDict = val->getDictFromJsonObject
+        valueDict
+        ->getArrayFromDict("queryData", [])
```

Comment:
in multiple places we are trying to extract the queryData.We need to make this generic